packages/bpk-theming/src/BpkThemeProvider.js

```diff
@@ -56,36 +55,26 @@ const createStyle = (theme, themeAttributes) => {
   return style;
 };
 
-class BpkThemeProvider extends Component {
-  getChildContext() {
-    return { theme: this.props.theme };
-  }
-
-  render() {
-    const {
-      children,
-      component: WrapperComponent,
-      style: userStyle,
-      theme,
-      themeAttributes,
-      ...rest
-    } = this.props;
+const BpkThemeProvider = (props) => {
```

Comment:
We change this to be a functional component as the `getChildContext` is deprecated and will be removed in a future version.